components/prism-bqn.js

```diff
@@ -0,0 +1,61 @@
+Prism.languages.bqn = {
+	'shebang': {
+	pattern: /^#!\s*\/.*/,
+	alias: 'important'
+	},
+	'comment': /#.*$/m,
```

Comment:
`$` (with the m flag present) is defined as `(?!.)`, so it will always trivially accept. It's essentially an unnecessary condition and can be removed.

```suggestion
	'comment': /#.*/,
```

Reply:
Also, please make `comment` greedy.